components/base/Typography.tsx

```diff
@@ -136,6 +137,50 @@ export const listStyles = css`
   }
 `;
 
+// Table styles
+export const tableStyles = css`
```

Comment:
Could we export these from `Table.tsx` instead since they're not really typography? I did something similar with `imageStyles` not too long ago

Reply:
Related, the current styling implemented in that file for the component version of tables is different than what's here. Can we make them consistent with each other?